src/main/kotlin/blackjack/Player.kt

```diff
@@ -0,0 +1,16 @@
+package blackjack
+
+class Player(val name: String = "", round: Round = Round()) {
+    private val _cards = mutableListOf(round.getCard(), round.getCard())
+    val cards: List<Card> get() = _cards
```

Comment:
```suggestion
    val cards: List<Card> get() = _cards.toList()
```
지금도 좋지만, 방어적 복사를 활용하시면 외부에서 원본을 손상시키지 못하게 막을 수 있습니다. 😄

Reply:
확인 부탁드립니다. 🙏